docs/user-guide/installation.rst

```diff
@@ -279,17 +279,12 @@ Server
       ``PULP_CONCURRENCY - 1``. For example, you can use ``sudo systemctl status pulp_worker-1`` to
       see how the second worker is doing.
 
-#. There are two more services that need to be running, but it is important that these two only run
-   once each (i.e., do not enable either of these on any more than one Pulp server).
-
-   .. warning::
-
-      ``pulp_celerybeat`` and ``pulp_resource_manager`` must both be singletons, so be sure that you
-      only enable each of these on one host if you are Pulp's clustered deployment.
+#. There are two more services that need to be running.
 
    On some Pulp system, configure, start and enable the Celerybeat process. This process performs a
    job similar to a cron daemon for Pulp. Edit ``/etc/default/pulp_celerybeat`` to your liking, and
-   then enable and start it. Again, do not enable this on more than one host. For Upstart::
+   then enable and start it. Multiple instances of this may be enabled, which will make the system
```

Comment:
s/this/pulp_celerybeat/
s/enabled/run concurrently/
s/the system/the Pulp installation/